src/main/java/model/Product.java

```diff
@@ -9,6 +9,12 @@
     public Product() {}
 
     public Product(long id, String name, String description) {
+        try {
+
+        } catch(Exception e) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either remove or fill this block of code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00108)